<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LVNNG7C" failures="3" tests="21" name="Test" time="167.659" errors="0" timestamp="2023-12-14T23:50:08 EST">
  <testcase classname="com.qa.testCasesTn.AddtoCartTest" name="checkingOutValidProduct" time="8.011">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.testCasesTn.AddtoCartTest.checkingOutValidProduct(AddtoCartTest.java:52)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkingOutValidProduct -->
  <testcase classname="com.qa.testCasesTn.SearchProductTest" name="verifySearchWithValidProduct" time="0.803"/>
  <testcase classname="com.qa.testCasesTn.SearchProductTest" name="verifySearchWithInvalidProduct" time="0.529"/>
  <testcase classname="com.qa.testCasesTn.SearchProductTest" name="verifySearchWithNoProduct" time="0.418"/>
  <testcase classname="com.qa.testCasesTn.RegisterTest" name="verifyingRegisterWithMandatoryFields" time="1.299"/>
  <testcase classname="com.qa.testCasesTn.RegisterTest" name="verifyingRegisterWithAllFields" time="1.173"/>
  <testcase classname="com.qa.testCasesTn.RegisterTest" name="verifyingRegisterWithExistingEmail" time="0.827"/>
  <testcase classname="com.qa.testCasesTn.RegisterTest" name="verifyingRegisterWithMismatchPassword" time="0.834"/>
  <testcase classname="com.qa.testCasesTn.RegisterTest" name="verifyingRegisterWithNoFields" time="0.616"/>
  <testcase classname="com.qa.testCasesTn.LoginTest" name="verifyLoginWithValidCredentials" time="1.162"/>
  <testcase classname="com.qa.testCasesTn.LoginTest" name="verifyLoginWithValidCredentials" time="1.116"/>
  <testcase classname="com.qa.testCasesTn.LoginTest" name="verifyLoginWithValidCredentials" time="0.916"/>
  <testcase classname="com.qa.testCasesTn.LoginTest" name="verifyLoginWithValidCredentials" time="0.985"/>
  <testcase classname="com.qa.testCasesTn.LoginTest" name="verifyLoginWithValidCredentials" time="0.902"/>
  <testcase classname="com.qa.testCasesTn.LoginTest" name="verifyLoginWithInvalidCredentials" time="0.639"/>
  <testcase classname="com.qa.testCasesTn.LoginTest" name="verifyLoginWithValidEmailInvalidPassword" time="0.612"/>
  <testcase classname="com.qa.testCasesTn.LoginTest" name="verifyLoginWithInvalidEmailValidPassword" time="0.614"/>
  <testcase classname="com.qa.testCasesTn.LoginTest" name="verifyLoginWithNoCredentials" time="0.55"/>
  <testcase classname="com.qa.testCasesTn.CheckOutTest" name="checkoutAsNewUser" time="0.285">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.testCasesTn.CheckOutTest.checkoutAsNewUser(CheckOutTest.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkoutAsNewUser -->
  <testcase classname="com.qa.testCasesTn.CheckOutTest" name="checkOutAsGuestUser" time="12.465"/>
  <testcase classname="com.qa.testCasesTn.CheckOutTest" name="checkOutAsExistingUser" time="0.342">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.qa.testCasesTn.CheckOutTest.checkOutAsExistingUser(CheckOutTest.java:104)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkOutAsExistingUser -->
</testsuite> <!-- Test -->
